atomic_reactor/plugins/pre_bump_release.py

```diff
@@ -0,0 +1,263 @@
+            self.committer_email = author_email
+
+        self.push_url = push_url
+        if commit_message is not None:
+            self.commit_message = commit_message
+
+    def bump_local_file(self, parser, label_key, next_release):
+        """
+        Rewrite a local Dockerfile with an incremented Release label
+        """
+
+        # Find where in the file to put the next release
+        content = startline = endline = None
+        cmd = 'LABEL'
+        for candidate in [insn for insn in parser.structure
+                          if insn['instruction'] == cmd]:
+            splits = shlex_splits(candidate['value'])
+
+            # LABEL syntax is one of two types:
+            if '=' not in splits[0]:  # LABEL name value
+                # remove (double-)quotes
+                value = candidate['value'].replace("'", "").replace('"', '')
+                words = value.split(None, 1)
+                if words[0] == label_key:
+                    # Adjust label value
+                    words[1] = next_release
+
+                    # Now reconstruct the line
+                    content = " ".join([cmd] + words) + '\n'
+                    startline = candidate['startline']
+                    endline = candidate['endline']
+                    break
+            else:  # LABEL "name"="value"
+                for token in splits:
+                    words = token.split("=", 1)
+                    if words[0] == label_key:
+                        # Adjust label value
+                        words[1] = next_release
+                        n = splits.index(token)
+                        splits[n] = "=".join(map(quote, words))
+
+                        # Now reconstruct the line
+                        content = " ".join([cmd] + splits) + '\n'
+                        startline = candidate['startline']
+                        endline = candidate['endline']
+                        break
+
+        # We know the label we're looking for is there
+        assert content and startline and endline
+
+        # Re-write the Dockerfile
+        lines = parser.lines
+        del lines[startline:endline + 1]
+        lines.insert(startline, content)
+        parser.lines = lines
+
+    @staticmethod
+    def get_next_release(current_release):
+        try:
+            return str(int(current_release) + 1)
+        except ValueError:
+            isdigit = type(current_release).isdigit
+            first_nondigit = [isdigit(x) for x in current_release].index(False)
+            n = str(int(current_release[:first_nondigit]) + 1)
+            return n + current_release[first_nondigit:]
+
+    def bump(self, repo, branch):
+        # Look in the git repository
+        remote = repo.remotes['origin']
+        if self.push_url:
+            repo.config['push.default'] = 'simple'
+            remote.push_url = self.push_url
+            remote.save()
+
+        # Bump the Release label
+        label_key = 'Release'
```

Comment:
Good point. So, the `assert_labels` plugin defaults to requiring `Release` but is configurable: https://github.com/projectatomic/atomic-reactor/blob/master/atomic_reactor/plugins/pre_assert_labels.py.

I should probably make the name of the label configurable here.